2-standard-library/activityMonitor.js

```diff
@@ -0,0 +1,56 @@
+const childProcess = require('child_process');
```

Comment:
[minor]: One of the bestpractice is using the "node" prefix pattern to indicate the node related libraries, like this
`const childProcess = require('node:child_process');`

Reply:
Done